src/reconstruction/gapfill.jl

```diff
@@ -0,0 +1,125 @@
+"""
+gapfill(
```

Comment:
are there more specific ways to call the function? `gapfill` might be a bit too generic name if we decide to add another kind of gapfilling later. Maybe `minimum_extension_gapfill` or so?

Reply:
renamed it, there are other variants